tests/karma.js

```diff
@@ -1064,6 +1064,204 @@ beforeAll(function (done) {
           body: unauthorizedXmlResponseBody
         }
       }))
+    .then(() =>
+      provider.addInteraction({
+        uponReceiving: 'Try to share with a link using the wrong password',
```

Comment:
You can do something like this
 ```suggestion
         uponReceiving: 'create share POST request with invalid auth ',
```